public/js/main.js

```diff
@@ -12,8 +12,19 @@ var hideMessages = function hideMessages(){
   }, 1000);
 }
 
+var updateGoogleLink = function updateGoogleLink(){
+  var link = $('.js-jtinder li').eq(0).attr('data-google-link');
+  $('.js-google-link').attr('href', link);
+}
+
 function loadNewPerson() {
   $('.js-extra-people li:first-child').appendTo('.js-jtinder ul');
+
+  console.log( $('.js-jtinder ul').text() );
+  console.log( $('.js-jtinder li').eq(1).text() );
+
+  setTimeout(updateGoogleLink, 500);
```

Comment:
I've used `setTimeout` here because the card that was just categorised is still being animated by jtinder, and won't be removed from the DOM until that animation is finished. So, we wait until it's gone, and the *new* person is properly at the top of the pile.